damage_calculator.py

```diff
@@ -289,20 +292,31 @@ def calc_dps(client, god, build, enemy, enemy_build, enemy_level, level=20):
                  "Ichaival": 0, "Silverbranch Bow": 0, "Total Item Damage": 0}
 
     flag = False
-    for i in range(10):
+    for i in range(number_of_autos):
         if "The Executioner" in build:
-            if i <= 4:
+            if i < 5 and i > 0:
```

Comment:
test to make sure this functions the same in game, iirc in testing it functions correctly with i <= 4

Reply:
i think with i<=4 means that the first auto gets exe shred, but the first auto only applies the exe shred, so the second one gets the benefit, so for i>0 and i<5, first pen is applied after first auto, but not sure how to test cause dk how to run it heh

Reply:
yea but does the shred get applied before dmg in game, giving the first auto the 7% shred. im fairly sure it does, youd have to check in jungle practice

Reply:
checked in game, shred applied after first auto